pyk/src/pyk/ktool/__init__.py

```diff
@@ -9,5 +9,5 @@
     pretty_print_kast,
     unparser_for_production,
 )
-from .kprove import KProve, kprove
+from .kprove import KProve, _kprove
```

Comment:
We really shouldn't be encouraging the use of private members.

```suggestion
from .kprove import KProve
```

Reply:
We must.

Reply:
The `_` is to discourage that use.

Reply:
True, but placing it in `__init__` makes it look as if is part of the public interface.

Reply:
I'd rather make it a separate issue/PR. We're the only users at the moment, we can make changes as we go.

Reply:
I've updated the PR with this change, hiding those imports by default.